reference/extensions/hooks.rst

```diff
@@ -1,4 +1,172 @@
 .. _reference_extensions_hooks:
 
 Hooks
------
\ No newline at end of file
+-----
+
+The Conan hooks is a feature intended to extend the Conan functionalities to perform certain orthogonal operations,
+like some quality checks, in different stages of a package creation process, like pre-build and post-build.
+
+Hook structure
+--------------
+
+A hook is a Python function that will be executed at certain points of Conan workflow
+to customize the client behavior without modifying the client sources or the recipe ones.
+
+Hooks can implement any functionality: it could be Conan commands, recipe interactions
+such as exporting or packaging, or interactions with the remotes.
```

Comment:
```suggestion
```
It is not very clear what it means, and hooks cannot implement commands, or packaging or interaction with remotes. Better remove it